### PR TITLE
Appears to make grib example work again

### PR DESCRIPTION
**Pull request by martindurant**

### CODE REVIEW

File: fsspec_reference_maker/grib2.py [hunk 1]
```diff
@@ -187,16 +188,17 @@ def example_multi(filter={'typeOfLevel': 'heightAboveGround', 'level': 2}):
...
+    files = ['hrrr.t22z.wrfsfcf01.json',
+     'hrrr.t23z.wrfsfcf01.json',
+     'hrrr.t00z.wrfsfcf01.json',
+     'hrrr.t01z.wrfsfcf01.json',
+     'hrrr.t02z.wrfsfcf01.json',
+     'hrrr.t03z.wrfsfcf01.json',
+     'hrrr.t04z.wrfsfcf01.json',
+     'hrrr.t05z.wrfsfcf01.json',
+     'hrrr.t06z.wrfsfcf01.json']
+    mzz = MultiZarrToZarr(files, remote_protocol="s3", remote_options={"anon": True}
```
Comment (lsterzinger) [inline fsspec_reference_maker/grib2.py:202]:
```suggestion
    mzz = MultiZarrToZarr(files, remote_protocol="s3", remote_options={"anon": True},
```

File: fsspec_reference_maker/grib2.py [hunk 2]
```diff
@@ -187,16 +188,17 @@ def example_multi(filter={'typeOfLevel': 'heightAboveGround', 'level': 2}):
...
-    #  'hrrr.t03z.wrfsfcf01.json',
-    #  'hrrr.t04z.wrfsfcf01.json',
-    #  'hrrr.t05z.wrfsfcf01.json',
-    #  'hrrr.t06z.wrfsfcf01.json']
-    # mzz = MultiZarrToZarr(files, remote_protocol="s3", remote_options={"anon": True}
-    #                       concat_kwargs={"dim": 'time'})
-    # mzz.translate("hrrr.total.json")
+
+def example_combine():
+    files = ['hrrr.t22z.wrfsfcf01.json',
```
Comment (lsterzinger) [inline fsspec_reference_maker/grib2.py:193]:
```suggestion
    from fsspec_reference_maker.combine import MultiZarrToZarr
    files = ['hrrr.t22z.wrfsfcf01.json',
```

File: fsspec_reference_maker/grib2.py [hunk 3]
```diff
@@ -187,16 +188,17 @@ def example_multi(filter={'typeOfLevel': 'heightAboveGround', 'level': 2}):
...
+     'hrrr.t23z.wrfsfcf01.json',
+     'hrrr.t00z.wrfsfcf01.json',
+     'hrrr.t01z.wrfsfcf01.json',
+     'hrrr.t02z.wrfsfcf01.json',
+     'hrrr.t03z.wrfsfcf01.json',
+     'hrrr.t04z.wrfsfcf01.json',
+     'hrrr.t05z.wrfsfcf01.json',
+     'hrrr.t06z.wrfsfcf01.json']
+    mzz = MultiZarrToZarr(files, remote_protocol="s3", remote_options={"anon": True},
+                          concat_kwargs={"dim": 'time'})
```
Comment (lsterzinger) [inline fsspec_reference_maker/grib2.py:203]:
```suggestion
                          xarray_concat_args={"dim": 'time'})
```

File: fsspec_reference_maker/grib2.py (hunk 2 above)
Comment (martindurant) [reply fsspec_reference_maker/grib2.py:193]: Yep, this was only run in an interactive session, as you probably guessed.

**Comment by martindurant** [comment]
Works now?

**Comment by martindurant** [comment]
@lsterzinger , I added you to the repo, so you can directly push to my branch.

**Comment by martindurant** [comment]
```
In [14]: fs = fsspec.filesystem("reference", fo='hrrr.total.json', remote_options={'anon': True}, remote_protocol="s3", skip_instance_cache=True)

In [15]: m = fs.get_mapper("")

In [16]: ds = xr.open_dataset(m, engine='zarr', consolidated=False)

In [17]: ds.v.values.mean()
Out[17]: -1.0653266
```

This was a by-hand combine as I fixed most things on Friday. I'll post the json in gitter.

**Comment by lsterzinger** [comment]
Working now, forgot `remote_options={'anon':True}` in `fsspec.filesystem()` :upside_down_face:

**Comment by lsterzinger** [comment]
I think we're good to merge this, I'll create a separate PR to add these as unit tests

**Pull request merged**